pygmt/src/text.py

```diff
@@ -76,8 +76,21 @@ def text_(  # noqa: PLR0912
     Parameters
     ----------
     textfiles : str or list
-        A text data file name, or a list of file names containing 1 or more
-        records with (x, y[, angle, font, justify], text).
+        A file name or a list of file names containing one or more records.
+        Each record has the following columns:
+
+        * *x*: X coordinate or longitude
+        * *y*: Y coordinate or latitude
+        * *angle*: Angle in degrees counter-clockwise from horizontal
+        * *font*: Text size, font, and color
+        * *justify*: Two-character justification code
```

Comment:
Here we write currently `two-character justification code`. But we write
- below for the `justify` parameter `two-letter combination`.
- in the [colorbar example](https://www.pygmt.org/dev/gallery/embellishments/colorbar.html) `2-character combination`
- in the [text tutorial](https://www.pygmt.org/dev/tutorials/basics/text.html) `two-letter (order independent) code`
- etc.

I think it would be nice to formulate this consistently. This is a bit related to issue #2834. I will try to find time to look at this in more detail; currently busy with other things.

Reply:
Yes, we can simplify these descriptions if issue #2834 is addressed.

Reply:
Then maybe we leave these formulations as they are for now, and make it consistent via addressing #2834 🙂.